client/verta/verta/environment/_python.py

```diff
@@ -29,30 +31,44 @@ class Python(_environment._Environment):
         captured.
     env_vars : list of str, optional
         Names of environment variables to capture. If not provided, nothing will be captured.
+    apt_packages : list of str, optional
+        Apt packages to be installed alongside a Python environment
     _autocapture : bool, default True
         Whether to enable the automatic capturing behavior of parameters above.
 
     Attributes
     ----------
     requirements : list of str
         pip requirements.
+    apt_packages : list of str
+        apt packages to be installed alongside a Python environment
 
     Examples
     --------
     .. code-block:: python
 
         from verta.environment import Python
         env1 = Python(requirements=Python.read_pip_environment())
+        env1.apt_packages = ["python3-opencv"]
+
         env2 = Python(requirements=Python.read_pip_file("../requirements.txt"))
+
         env3 = Python(
             requirements=["tensorflow"],
             env_vars=["CUDA_VISIBLE_DEVICES"],
+            apt_packages=["python3-opencv"]
         )
-
     """
 
-    def __init__(self, requirements, constraints=None, env_vars=None, _autocapture=True):
-        super(Python, self).__init__(env_vars, _autocapture)
+    def __init__(
+        self,
+        requirements,
+        constraints=None,
+        env_vars=None,
+        apt_packages=None,
+        _autocapture=True,
+    ):
+        super(Python, self).__init__(env_vars, _autocapture, apt_packages=apt_packages)
```

Comment:
Passing through `apt_packages` to the parent __init__ and added docstring examples